### DISCUSSION TITLE
WebView is outdated

### DISCUSSION THREAD

**Discussion by ballo** [tutao/tutanota#5357] [Q&A]
- [x] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [x] I've searched and d
**Describe the bug**
Launching the app causes this error to show up. There is no work-around. Using the latest app checked on both Google Play and F-Droid.

**To Reproduce**
1. Open app
2. No step 2

**Expected behavior**
I expected the usual app with the usual bugs I have to work around. This has no work around. I can't access my email.

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: s8
 - OS: 9
 - Version 3.112.6

**Comment by ganthern** [comment]
I may be misunderstanding you, but we're using the web view provider that's installed on your device, so you'll have to update it if it is outdated.

**Reply by ballo** [reply]
What does that even mean?

**Reply by ganthern** [reply]
It's been a while, but for future readers: that means that we use the WebView (which is a program that understands javascript and html, the stuff websites are made of) to display our user interface. A WebView provider is a separate program/app that's installed on pretty much any android device that offers this functionality to other applications. You can choose which app provides the webview in the system settings.

**Comment by CubitOom** [comment]
As someone who prefers to keep webview disabled for security reasons. 

How much effort would it be to have an option on the android app to select what browser to use? Or to just use the default browser. 

I don't need webview for anything on my phone except for this app and I'd rather keep it disabled.

**Reply by ganthern** [reply]
We're using features of the webview that can not be provided by the system browser. You're asking for a setting that disables half the app.

You can use your default browser to access the web app, though. Maybe save a bookmark to the home screen?

E: Also, I know this will probably not ease your mind, but for others reading this: anything the webview can do to your system is filtered by the native part of our app, including network requests. With one exception: the mechanism the website is using to receive events from the server does not go through our native part, but is directly going from the webview to the network.
